bindings/nodejs/lib/utils/index.ts

```diff
@@ -4,3 +4,34 @@
 export * from './utf8';
 export * from './utils';
 export * from '../types/utils';
+
+/**
+ * Converts a byte array to a hexadecimal string.
+ *
+ * @param {Uint8Array} byteArray - The bytes to encode.
+ * @param {boolean} [prefix=false] - Whether to include the '0x' prefix in the resulting hexadecimal string.
+ * @returns {string} The hexadecimal representation of the input byte array.
+ */
+export const bytesToHex = (bytes: ArrayLike<number>, prefix = false) => {
```

Comment:
```suggestion
export const bytesToHex = (bytes: ArrayLike<number>, prefix = true) => {
```
Shouldn't the default be true? Since we always have it with prefix in the types?

Reply:
id like to, but firefly has it like this in the converter, so would be nicer for them to not have to change things right?

Reply:
arguably, hexToBytes removes the prefix, so the expected bytestohex input also has no prefix